lms/djangoapps/verify_student/models.py

```diff
@@ -698,7 +704,11 @@ def upload_photo_id_image(self, img_data):
             return
 
         aes_key = random_aes_key()
-        rsa_key_str = settings.VERIFY_STUDENT["SOFTWARE_SECURE"]["RSA_PUBLIC_KEY"]
+        
+        if settings.ENABLE_HOUSTON_PHOTO_VERIFICATIONS:
```

Comment:
the not using of microsite_configuration here, implies that this verification method will always be on for all clients if it is set to true, is that the case? do we want to make this the default for all?